src/components/material/material-buttons/material-buttons.test.ts

```diff
@@ -7,33 +7,25 @@ describe("Material buttons", () => {
       url: "**/availability/v3?recordid=**",
       fixtureFilePath: "material/unavailability.json"
     });
-    cy.visit("/iframe.html?id=apps-material--default&viewMode=story&type=bog")
-      .getBySel("material-description")
-      .scrollIntoView();
-
+    cy.visit("/iframe.html?id=apps-material--default&viewMode=story&type=bog");
+    cy.scrollTo("bottom");
```

Comment:
Scrolling across the entire page seems a bit odd to me compared to scrolling to the element which is responsible for the element. Perhaps it is necessary 🤔 .

Reply:
Yeah, unfortunately it is necessary, because the elements don't exist before they enter viewport. The test was failing for `.scrollIntoView()` 😿